branca/element.py

```diff
@@ -189,13 +189,13 @@ def __init__(self, url, download=False):
         if download:
             self.get_code()
 
-        self._template = Template(u"""
-        {% if kwargs.get("embedded",False) %}
-            <script>{{this.get_code()}}</script>
-        {% else %}
-            <script src="{{this.url}}"></script>
-        {% endif %}
-        """)
+        self._template = Template(
+            '{% if kwargs.get("embedded",False) %}'
+            '<script>{{this.get_code()}}</script>'
+            '{% else %}'
+            '<script src="{{this.url}}"></script>'
+            '{% endif %}'
```

Comment:
Mixed of double quotes and single quotes. I would stick to single (when possible) as JSON enforces double.  That way I know when I am writing Python and when I am writing JSON :wink: